tests/test_people_issue_ar170.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3 -B
+import unittest
+
+from cromulent import vocab
+
+from tests import TestPeoplePipelineOutput, classification_sets, classification_tree, classified_identifiers
+
+vocab.add_attribute_assignment_check()
+
+class PIRModelingTest_AR170(TestPeoplePipelineOutput):
+    '''
+    AR-170: Identify generic group records using the generic field name and populate profesional activity accordingly
+    '''
+    def test_modeling_ar170(self):
+        output = self.run_pipeline('ar170')
+
+        groups = output['model-groups']
+
+        expected = {
+            'tag:getty.edu,2019:digital:pipeline:REPLACE-WITH-UUID:shared#GROUP,AUTH,Strubin' : {
+                'label' : "Professional activity of persons in the 18th century",
```

Comment:
The labels in many of these records seem either wrong or misleading. This record is for "Strubin", *not* for any "persons in the 18th century".

I think the code that creates these labels needs some fixes to ensure these generic "persons" labels are only created for the anonymous records (the ones that used to be matched by the regex patterns containing nationalities and/or centuries).